org.knime.knip.io/src/org/knime/knip/io/nodes/imgreader2/readfrominput/ImgReaderTableNodeModel.java

```diff
@@ -156,9 +156,8 @@ public ImgReaderTableNodeModel() {
 
 				if (dataRow.getSecond().isPresent()) {
 					encounteredExceptions.set(true);
-					LOGGER.debug("Encountered exception while reading image " + dataRow.getFirst().getKey()
-							+ "! Caught Exception: " + dataRow.getSecond().get().getMessage());
-					LOGGER.debug(dataRow.getSecond().get());
+					LOGGER.warn("Encountered exception while reading image " + dataRow.getFirst().getKey()
```

Comment:
Actually I've set them to debug on purpose. You don't want to get spammed with warnings. You can look at the log if you want more information.